src/Taxjar/TaxjarApi.cs

```diff
@@ -118,6 +118,10 @@ protected virtual RestRequest CreateRequest(string action, Method method = Metho
                             request.AddQueryParameter(prop.Name, prop.Value.ToString());
                         }
                     }
+                    else
```

Comment:
This could potentially be a breaking change, except for the fact that if we're doing this our API does not support a body outside of the normal flows? Just thinking what we should list this as version wise for semver.

Reply:
Discussed elsewhere; RestSharp definitely introduced a breaking change by disallowing bodys for GET requests. However, absent abuse of our client nothing should change for developers with normal usage. Therefore, we'll publish a minor SemVer release.